sessions/006/README.md

```diff
@@ -580,11 +580,15 @@ This example has some tricky steps highlighted in bold, where you need to be ext
 
 First, [deploy the Strimzi Cluster Operator and Kafka cluster](/sessions/001).
 
-When the cluster is ready, we break 2 ZooKeeper servers by writing corrupted data into ZK log.
+When the cluster is ready, we create a topic first so that it can be used to make sure the data is not lost after recovery.
+And then break 2 ZooKeeper servers by writing corrupted data into ZK log.
 
 ```sh
 $ CLUSTER_NAME="my-cluster"
 
+$ kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --create --topic t0
+Created topic t0.
```

Comment:
Should we add describe after this, and at the end show that we have the same topicId?

Reply:
Nice suggestion! Updated.